src/Compilers/Test/Core/Compilation/CompilationExtensions.cs

```diff
@@ -4,7 +4,7 @@
 
 #nullable disable
 // Uncomment to enable the IOperation test hook on all test runs. Do not commit this uncommented.
-//#define ROSLYN_TEST_IOPERATION
+// #define ROSLYN_TEST_IOPERATION
```

Comment:
It doesn't look like there are meaningful changes in this file. Consider reverting.

Reply:
This does not appear to be addressed.